hive/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -63,13 +60,13 @@
 public class HiveTableOperations extends BaseMetastoreTableOperations {
```

Comment:
@rdblue do we need to keep the comment about thread-safety on line 57? The pool should resolve this issue.

Reply:
You're right. I've removed it.